doc/sphinxman/source/detci.rst

```diff
@@ -135,6 +128,38 @@ which the driver has built-in shortcuts, so long as the relevant options
 examples of single-refence CI, RASCI, and full CI computations are provided
 in :source:`samples`.
 
+.. _`table:ci_spaces`:
+
+.. table:: Orbital spaces for CI computations
+
+    +----------------------------+----------------------------+----------------------------+
+    | CI (e.g., CISD, FCI)       | RASCI                      | CASCI                      |
+    +============================+============================+============================+
+    | |globals__frozen_uocc|     | |globals__frozen_uocc|     |                            |
+    +----------------------------+----------------------------+                            +
+    | (all orbitals not in       | |globals__ras4|            |                            |
+    + |globals__frozen_uocc|     +----------------------------+                            +
+    | or |globals__frozen_docc|  | |globals__ras3|            |                            |
+    + are included in CI)        +----------------------------+                            +
+    |                            | |globals__ras2|            |                            |
+    +                            +----------------------------+                            +
+    |                            | |globals__ras1|            | |globals__active|          |
+    +----------------------------+----------------------------+----------------------------+
+    | |globals__frozen_docc|     | |globals__frozen_docc|     | |globals__frozen_docc|     |
+    +----------------------------+----------------------------+----------------------------+
+
+The table above shows the relevant orbitals spaces for CI computations (an
+analogous :ref:`table <table:mcscf_spaces>` for MCSCF is also available).  
+The third column of the
+table refers to CASCI, in which a full CI is performed in some smaller
+set of ``active`` orbitals; it is equivalent to CASSCF except without
+any orbital optimization.  It can be invoked via ``energy('fci')``
+with appropriate values selected for |globals__frozen_docc| and
+|globals__active|.  For CI computations, there is no difference between
+|globals__frozen_docc| and |globals__restricted_docc|, or between
+|globals__frozen_uocc| and |globals__restricted_uocc|.  There are
+differences betwen these keywords for :ref:`sec:mcscf`.
```

Comment:
Working on this again today.... can't spot a typo in the above fragment... what's the typo?